README.md

```diff
@@ -1,16 +1,201 @@
+* Create an ACAT report(optional): Go to Azure Portal to create an ACAT report for you application.
+* Prepare the deployment id(optional): You can also get quick assessment by your deployment, set the deployment id as output in your former deploy action, and take the deployment id as input of Get Quick Assessments action.
+
+
+
+# Inputs for the Action
+
+* `cred`: mandatory. The credential you use to get quick assessments.
+* `report-name`: Optional. If you want to get quick assessments by report, you should create a report before you run the github action and set the report-name value the name of the report you created. At least one of the 2 parameters `report-name` and `deployment-id` must be filled.
+* `deployment-id`: Optional. If you want to get quick assessments by deployment, you should get the id of your deployment, and pass the value to `deployment-id`. At least one of the 2 parameters `report-name` and `deployment-id` must be filled.
+
+
+# End-to-End Sample Workflows
+
+  
+### Sample workflow to apply all  policy file changes in a given directory to Azure Policy
+
+
+```yaml
+# File: .github/workflows/workflow.yml
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Test artifact
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Azure login
+      uses: Azure/login@v1.4.6
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+    - name: analyse Microsoft 365 compliance results
+      uses: ./ # Uses an action in the root directory
+      with:
+        cred: ${{ secrets.AZURE_CREDENTIALS }}
+        report-name: 'test-report'
+        
+```
+The above workflow will get assessments by report.
+
+
+### Sample workflow to apply only a subset of assignments from a given directory to Azure Policy
+
+
+```yaml
+# File: .github/workflows/workflow.yml
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Test artifact
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Azure login
+      uses: Azure/login@v1.4.6
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS }}
+    - name: Deploy with ARM template
+      id: deployarm
+      uses: azure/arm-deploy@v1
+      with:
+        subscriptionId: ${{ secrets.SUBSCRIPTION_ID }}
+        resourceGroupName: ${{ secrets.RESOURCE_GROUP}}
+        template: ./deploy/Storage.template.json
+        parameters: storageAccountType=Standard_LRS
+    - run: echo ${{ steps.deployarm.outputs.deploymentId }}
+
+    - name: analyse Microsoft 365 compliance results
+      uses: ./ # Uses an action in the root directory
+      with:
+        cred: ${{ secrets.AZURE_CREDENTIALS }}
+        deployment-name: ${{ steps.deployarm.outputs.deploymentId }}
+        
+```
+The above workflow will get quick assessments by deployment. 
+
+
+
+
+
+
+# Configure credentials for Azure login action:
```

Comment:
Shall we place this section in `prerequisites`? BTW I remember this part is covered by Github doc, maybe we can just leave a link here.